pkg/trace/otel.go

```diff
@@ -226,16 +226,27 @@ func (t *otelTracer) end(ctx context.Context) {
 func (t *otelTracer) addInfo(ctx context.Context, args ...log.Marshaler) {
 	if span := trace.SpanFromContext(ctx); span != nil {
 		for _, arg := range args {
+			if arg == nil {
```

Comment:
I don't think `arg` will ever be `nil`.
https://go.dev/play/p/r7lBirpAWCM

Reply:
thanks!